### ISSUE TITLE
No Homebrew on Ubuntu

### ISSUE THREAD

**Issue by fredRos** [goedman/Stan.jl#37]
I'm running julia 0.5 on Ubuntu 16.04. Installing CmdStan was no problem but I got this error with Stan.jl

```julia
julia> Pkg.add("Stan")
INFO: Cloning cache of Stan from https://github.com/goedman/Stan.jl.git
INFO: Installing Stan v1.0.2
INFO: Building Rmath
INFO: Building Cairo
INFO: Building Stan
==========================================================[ ERROR: Stan ]==========================================================

LoadError: ArgumentError: Module Homebrew not found in current path.
Run `Pkg.add("Homebrew")` to install the Homebrew package.
while loading /home/beaujean/.julia/v0.5/Stan/deps/build.jl, in expression starting on line 1

===================================================================================================================================

=========================================================[ BUILD ERRORS ]==========================================================

WARNING: Stan had build errors.

 - packages with build errors remain installed in /home/beaujean/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("Stan")`
 - build a single package by running its `deps/build.jl` script

===================================================================================================================================
INFO: Package database updated
```

Now I read the Readme that says Stan.jl is mostly tested on Mac but why would you require Homebrew for any platform? I guess it only makes sense to have on Mac OS. 

Since your `build.jl` should do nothing if not on Mac OS, I guess you just have to avoid the first line

```julia
using BinDeps, Homebrew
```
that requires homebrew regardless of the operating system.

**Comment by goedman** [comment]
Thank you Frederiek. I'll will try to fix this asap, hopefully later today. Or if you have a PR, I'll review the suggested fixes.

Thanks again,
Rob

**Comment by fredRos** [comment]
Thanks Rob. I can look into a PR in about a week. I"m not a rush, so no
worries. This way I can test it on a linux machine to which you seem to
not have access

Fred

**Comment by goedman** [comment]
Hi Frederik,

Is Stan.jl installed on your system after showing below error message? Does Pkg.test(“Stan”) work?

Can you try Pkg.checkout(“Stan”) to get the latest master?

As far as I can see Homebrew should not be REQUIREd nor used in the latest version:

in REQUIRE:

…
@osx Homebrew 0.4.0


in build.jl:

if is_apple()
	…
end

I’m not 100% sure this was already the case for Stan v”1.0.2”.

Regards,
Rob

**Comment by fredRos** [comment]
Yes you are right, this is fixed in d9c4e6e0cb7749ef31259f569f0fa98b3cf104de so the latest master has it but the latest official release 1.0.2 does not. If it's not much trouble, why not publish 1.0.3?

After Pkg.checkout("Stan"), the pkg builds and the tests pass!

**Comment by goedman** [comment]
Hi Fred, yes, I should tagged 1.0.3. It usually takes me a day before all is set and done to tag and publish a new version, hence my reluctance :-(.

I might hold off a bit longer for a Julia v"0.6" versions of Mamba and Gadfly. If that takes too long, say 1 month, I'll publish an intermediate tag.

Regards,
Rob
